src/index.html

```diff
@@ -9,11 +9,25 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>Enclosures</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <div class="outBox redBorder" id="qa-block">
+    <div class="midBox redBorder">
+      <div class="inBox redBorder" >In programming languages, a closure is a technique for implementing lexically scoped name binding in a language with first-class functions. Operationally, a closure is a record storing a function together with an environment.</div>
```

Comment:
It's simple text, use `p` instead of `div`

Reply:
тоже странное замечание, у тех кого заапрувил ты, тоже везде дивы стоят, но к ним претензий нет

Reply:
where did you see approve work with `div` instead of `p`?
![Знімок екрана  о 3 33 55 пп](https://user-images.githubusercontent.com/44294945/63940565-720c4500-ca72-11e9-8e70-1ea61b2ebdbd.png)

Reply:
да, в нашей половине группы таких нет, вот я и спрашиваю почему другие та апрувают такое решение,но тебя такое не устраивает?